### ISSUE TITLE
无法使用拖拽功能

### ISSUE THREAD

**Issue by ZiuChen** [universeroc/super-drag#5]
MacOS Monterey 12.4
Chrome 110.0.5481.77 (x86_64) 

Worker报错:
```
Error handling response: TypeError: Cannot read properties of undefined (reading 'length')
    at chrome-extension://dlknjglebgomjjfaijjnebecgjbfjihk/service-worker.js:110:28
Unchecked runtime.lastError: Invalid url pattern 'http*://*/*'
```

**Comment by universeroc** [comment]
> MacOS Monterey 12.4 Chrome 110.0.5481.77 (x86_64)
> 
> Worker报错:
> 
> ```
> Error handling response: TypeError: Cannot read properties of undefined (reading 'length')
>     at chrome-extension://dlknjglebgomjjfaijjnebecgjbfjihk/service-worker.js:110:28
> Unchecked runtime.lastError: Invalid url pattern 'http*://*/*'
> ```

收到，会在下个版本修复上线，感谢反馈

**Comment by universeroc** [comment]
fixed at 0.9.1 already in publishing process
